src/Qwiq.Core/IQueryDefinition.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Microsoft.Qwiq
+{
+    public interface IQueryDefinition : IIdentifiable<Guid>, IEquatable<IQueryDefinition>
```

Comment:
What about URI?

Reply:
I'll add uri.

Reply:
Err...So Uri doesn't exist for soap, we can either do NotSupported for SOAP or add additional IQueryDefinition for Rest with Rest only property.